### DISCUSSION TITLE
Utilizzo di use_gross_prices ed arrotondamenti.

### DISCUSSION THREAD

**Discussion by francescoRubini** [fattureincloud/api#261] [Q&A] [answered]
Buongiorno.

Ho notato un errore nel calcolo dell'arrotondamento in caso di scorporo dell'iva.

Se invio una fattura (con impostato use_gross_prices = true) un importo di 82,00€ con iva 5% il sistema calcola
imponibile 78,10€ ed iva 3,90€. 

Ma non è corretto.

l'iva di 78,10 è 3,905 quindi 3,91€. Ed il totale sarebbe quindi 82,01€

E' stato tolto un centesimo all'IVA per fa quadrare, ma questo è sbagliato (oltre che evasione di iva)

Dovrebbe essere tolto un centesimo all'imponibile per far "quadrare" poi l'importo totale della fattura.

Quindi il risultato dovrebbe essere (78,10 di imponibile - 0,01 di arrotondamento) e 3,91€ di iva.

Se, invece, quanto da voi calcolato è fiscalmente corretto, avete una "pezza" giustificativa da girare ai commercialisti che ci hanno fatto notare l'errore?

Vi ringrazio in anticipo.

**Comment by valmoz** [comment, marked as answer]
Buongiorno @francescoRubini 
chiedo scusa per il ritardo.

Le consiglio di contattare il supporto clienti sulla pagina di Fatture in Cloud.

Grazie
Mauro

**Reply by francescoRubini** [reply]
ok, lo farò.

Grazie,
Francesco
